Add Cape Verde and Somalia into countries

rileywaffle opened the pull request:
If you have any questions, msg me here or on discord @ rileywaffle

rileywaffle committed:
Add Somalia and Cape Verde

rileywaffle committed:
Merge branch 'main' of https://github.com/rileywaffle/osu-world-maps

agatemosu commented on src/countries/Somalia.ts:
```diff
@@ -0,0 +1,30 @@
+    1720057, // Gedo
+    1720058, // Hiiraan
+    1720061, // Jubbada Dhexe
+	1720059, // Jubbada Hoose
+	1720063, // Mudug
+	1720064, // Nugaal 
+	13316065, // Sahil 
+	1720065, // Sanaag 
+	1720062, // Shabeellaha Dhexe 
+	1720060, // Shabeellaha Hoose 
+	1720066, // Sool
+	1720067, // Togdheer	
+	1720068, // Woqooyi Galbeed
+  ];
+}
```
```suggestion
    1720059, // Jubbada Hoose
    1720063, // Mudug
    1720064, // Nugaal 
    13316065, // Sahil 
    1720065, // Sanaag 
    1720062, // Shabeellaha Dhexe 
    1720060, // Shabeellaha Hoose 
    1720066, // Sool
    1720067, // Togdheer	
    1720068, // Woqooyi Galbeed
  ];
}

```

rileywaffle committed:
Update src/countries/Somalia.ts

Co-authored-by: Ana <agaatem@outlook.com>

rileywaffle committed:
Create Bolivia.ts

rileywaffle commented:
Also added Bolivia.

rileywaffle committed:
Added Multiple Counties

Added Costa Rica, Croatia, Dominican Republic, Egypt, Morocco, Slovenia, UAE, and Uruguay.

rileywaffle commented:
Added 7 more counties above.

rileywaffle committed:
Added Multiple Countries

Added Alergia, Cambodia, Guatemala, Mongolia, and Tunisia

rileywaffle commented:
Added 5 more counties above.

rileywaffle committed:
Update CostaRica.ts

rileywaffle committed:
Update Uruguay.ts

rileywaffle committed:
Update UnitedArabEmirates.ts

rileywaffle committed:
Update Slovenia.ts

rileywaffle committed:
Update Morocco.ts

rileywaffle committed:
Update DominicanRepublic.ts

rileywaffle committed:
Update Egypt.ts

rileywaffle committed:
Added Multiple Countries

Added Bahrain, Bangladesh, Bosnia and Herzegovina, Brunei, El Salvador, Georgia, Honduras, Iraq, Jordan, Kuwait, Kyrgyzstan, Myanmar, Nepal, North Macedonia, Pakistan, Paraguay, Qatar, and Trinidad and Tobago

rileywaffle commented:
Added 18 more, finished all WIP's that were complete.

uzervlad committed:
Fix indents

uzervlad committed:
Fix North Macedonia IDs

uzervlad committed:
Fix Slovenia IDs

uzervlad committed:
Fix Georgia IDs

uzervlad committed:
Fix Somalia IDs

uzervlad committed:
Fix Cabo Verde name

rileywaffle committed:
Added more countries

Added Albania, Armenia, Iceland, Laos, Lebanon, Nicaragua, Oman, and Sri Lanka

rileywaffle commented:
added a few more countries

rileywaffle committed:
Added more countries

Added Afghanistan, Andorra, Angola, Barbados, Burkina Faso, and Uzbekistan.

uzervlad committed:
Fixes

uzervlad commented:
Merging for now, but Cabo Verde division doesn't seem to match what I could find, so I'll skip it this release.
